firefly_client/firefly_client.py

```diff
@@ -595,46 +597,77 @@ def show_table(self, file_on_server, tbl_id=None, title=None, page_size=100, is_
+            A table ID. It will be created automatically if not specified.
+        page_size : int, optional
+            The number of rows to fetch.
 
-        Other Parameters
+        Returns
+        -------
+        out : dict
+            Status of the request, like {'success': true}. 
+        """
+
+        if not tbl_id:
+            tbl_id = FireflyClient._gen_item_id('Table')
+        tbl_req = {'startIdx': 0, 'pageSize': page_size, 'source': file_on_server,
+                   'id': 'IpacTableFromSource', 'tbl_id': tbl_id}
+        meta_info = {'title': tbl_id, 'tbl_id': tbl_id}
+        tbl_req.update({'META_INFO': meta_info})
+        payload = {'request': tbl_req, 'hlRowIdx': 0}
+        return self.dispatch_remote_action(self.channel, FireflyClient.ACTION_DICT['FetchTblData'], payload)
+
+    def show_xyplot(self, tbl_id, chart_params, standalone=False):
+        """
+        Show a XY plot
+
+        Parameters
+        ----------
+        tbl_id : str, required
+            A table ID of the data to be plotted.
+        chart_params
+            parameters for XY Plot, please see the details in 'Chart Parameters'.
+        standalone : bool, optional
+            When true, the chart is always present in the chart area, 
+            no matter if the related table is present or not 
+
+        Chart Parameters
         ----------------
-        source: str
-            location of the ipac table, url or file path; ignored when XY plot view is added to table.
-        chartTitle : str
-            title of the chart.
-        xCol: str
+        chart_params.xCol: str
             column or expression to use for x values, can contain multiple column names,
             ex. log(col) or (col1-col2)/col3.
-        yCol: str
+        chart_params.xError: str
+            column or expression to use for x error, can contain multiple column names
+        chart_params.yCol: str
             column or expression to use for y values, can contain multiple column names,
             ex. sin(col) or (col1-col2)/col3.
-        xyRatio : numeric types
+        chart_params.yError: str
+            column or expression to use for x error, can contain multiple column names
+        chart_params.xyRatio : numeric types
             Aspect ratio (must be between 1 and 10).
-        stretch : {'fit', 'fill'}
+        chart_params.stretch : {'fit', 'fill'}
             Stretch method.
-        xLabel : str
+        chart_params.xLabel : str
             label to use with x axis.
-        yLabel : str
+        chart_params.yLabel : str
             label to use with y axis.
-        xUnit : str
+        chart_params.xUnit : str
             unit for x axis.
-        yUnit : str
+        chart_params.yUnit : str
             unit for y axis.
-        xOptions : str
+        chart_params.xOptions : str
             Comma separated list of x axis options: grid,flip,log.
-        yOptions : str
+        chart_params.yOptions : str
```

Comment:
from `chart_params.xCol` to here, please remove the `chart_params.` inside the docstring.